src/germline.cpp

```diff
@@ -15,12 +15,14 @@ namespace linearham {
 /// sites.
 /// @param[in] next_transition
 /// Vector of probabilities of transitioning to the next match state.
+/// @param[in] scaler
+/// An underflow scaler.
 Germline::Germline(Eigen::VectorXd& landing, Eigen::MatrixXd& emission_matrix,
-                   Eigen::VectorXd& next_transition)
-    : emission_matrix_(emission_matrix) {
+                   Eigen::VectorXd& next_transition, double& scaler)
```

Comment:
Doubles are always the same size, irrespective of the size of the number. There's no performance problem passing a double, and the reference may just confuse people thinking that you intend to modify the value of scaler internally.